.evergreen.yaml

```diff
@@ -20,6 +20,22 @@ functions:
         working_dir: src/atlas-service-broker
         script: |
           go vet ./...
+  "setup_snyk":
+    - command: shell.exec
+      type: system
+      params:
+        script: |
+          curl -L -o snyk https://github.com/snyk/snyk/releases/download/v1.216.0/snyk-linux
+          chmod +x snyk
+          mv snyk /usr/local/bin/
+  "snyk_snapshot":
+    - command: shell.exec
+      type: system
+      params:
+        working_dir: src/atlas-service-broker
+        script: |
+          snyk auth ${snyk_token}
```

Comment:
Added this as "snyk-evergreen-upload-atlas-service-broker" 

It already had this name, so I wanted to be consistent.

Reply:
Sorry, what are you referring to with "snyk-evergreen-upload-atlas-service-broker"?

Reply:
Updated the name of the expansion to "snyk-evergreen-upload-atlas-service-broker".